mkdocs_git_revision_date_localized_plugin/plugin.py

```diff
@@ -121,6 +125,12 @@ def on_page_markdown(
             str: Markdown source text of page as string
         """
 
+        excluded_titles = self.config.get("excluded_page_titles", [])
+        if page.title in excluded_titles:
+            logging.debug("Excluding page " + page.url)
```

Comment:
I'm going to change this bit to use `page.file.src_path` instead of `page.title`.

That's because you can define the title in both the mkdocs.yml file and in the markdown itself. `page.file.src_path` is more straightforward and more decoupled from the content. To ignore the index page for example, you specify `index.md`.

Reply:
That's true.  Sounds better then my approach.